src/worker/WorkerBridge.js

```diff
@@ -12,7 +12,7 @@ class WorkerBridge {
     }
 
     pollQueue(){
-        self.setTimeout(() => {
+        setTimeout(() => {
```

Comment:
Perf can be much better if we don't send operations periodically, but instead have some sort of feedback from the part that does the rendering. Also, for different targets, i noticed that value of TIMEOUT gives better frame rates